src/pages/utilities/UnrecognizedUtilityTabs/UnrecognizedTab.tsx

```diff
@@ -299,51 +224,36 @@ function UnrecognizedTab() {
     getFilteredRowModel: getFilteredRowModel(),
   });
   const tableSelectedRows = table.getSelectedRowModel();
-
-  useEffect(() => {
-    table.resetRowSelection();
-  }, [files, table]);
-
-  const selectedRows = useMemo(
-    () => tableSelectedRows.rows.map(row => row.original),
-    [tableSelectedRows],
-  );
-
-  const isAvdumpFinished = useMemo(() => {
-    let notFound = false;
-    forEach(selectedRows, (row) => {
-      if (!avdumpList[row.ID]?.hash) {
-        notFound = true;
-        return false;
-      }
-      return true;
-    });
-    return !notFound;
-  }, [selectedRows, avdumpList]);
-
-  const avdumpFiles = async () => {
-    for (let i = 0; i < selectedRows.length; i += 1) {
-      // Files cannot be dumped in parallel on the server, yet
-      // Use Promise.all when avdump3 is implemented and parallel dumping is possible
-      if (!avdumpList[selectedRows[i].ID]?.hash) {
-        // eslint-disable-next-line no-await-in-loop
-        await runAvdump(selectedRows[i].ID);
+  const selectedRows = useMemo(() => tableSelectedRows.rows.map(row => row.original), [tableSelectedRows]);
+  const isAvdumpFinished = useMemo(() => every(selectedRows, row => row.AVDump.LastDumpedAt), [selectedRows]);
+  const dumpInProgress = some(avdumpList.sessions, session => session.status === 'Running');
```

Comment:
Note: This is a good place to use the `some` function. The other place is not it